packages/react-scripts/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=20.0.0"
+    "node": ">=24.0.0"
```

Comment:
I'm thinking about this change...I wonder if this PR should be one of the last PRs to get merged concerning node 24. Because if an app updates to react-scripts 8.11.0 but they haven't updated their app's version of node yet, then I think the npm install step would fail. Which is probably friction we don't want to introduce.

let's bring this up in standup tomorrow morning.